### PR TITLE
Adds separate key for aggregator signatures

### PR DESCRIPTION
**Pull request by mauriceLC92**
The following PR adds a separate private key for aggregator signatures when creating a new node.

### CODE REVIEW

**Comment by mauriceLC92** [comment]
@tzdybal I have left you some messages on Discord too but have created draft PR for comments.

Not sure if I have missed the mark on what was required for this issue so please let me know :)

**Review by tzdybal — COMMENTED** [review]
Thanks for your submission! This looks good!
I would only rename `managerKey` to `signingKey` or `proposerKey` or `aggregatorKey`.

File: node/node.go [hunk 1]
```diff
@@ -78,7 +78,7 @@ type Node struct {
 }
 
 // NewNode creates new Optimint node.
-func NewNode(ctx context.Context, conf config.NodeConfig, nodeKey crypto.PrivKey, clientCreator proxy.ClientCreator, genesis *tmtypes.GenesisDoc, logger log.Logger) (*Node, error) {
+func NewNode(ctx context.Context, conf config.NodeConfig, p2pKey crypto.PrivKey, managerKey crypto.PrivKey, clientCreator proxy.ClientCreator, genesis *tmtypes.GenesisDoc, logger log.Logger) (*Node, error) {
```
[the remark this answers is not in the document]
Comment (mauriceLC92) [reply node/node.go]: Have now updated

**Comment by tzdybal** [comment]
Updated the branch and changed from "improvement" to "breaking change" in CHANGELOG-PENDING.md as this changes external API used in Cosmos-SDK / Ethermint.

**Review by tzdybal — APPROVED** [review]
I'll merge the changes after releasing v0.1.1 (#316).

**Comment by adlerjohn** [comment]
This should be good to go now @tzdybal?

**Comment by tzdybal** [comment]
Yes, I'll merge ASAP.

**Pull request merged**